hipster.pkr.hcl

```diff
@@ -82,7 +87,7 @@ source "qemu" "oi-hipster" {
   disk_size = local.disk_size
   iso_checksum = var.iso_checksum
   iso_url =  local.iso_url
-  iso_checksum_type = local.iso_checksum_type
+  //iso_checksum_type = local.iso_checksum_type
   shutdown_command = local.shutdown_command
```

Comment:
Can you add a comment why it has been commented out? Either this or completely remove the lines.